cli/cmd/develop.go

```diff
@@ -70,3 +87,21 @@ func checkForgeInstalled() bool {
 
 	return err == nil // If there is no error, forge is installed.
 }
+
+// sanitizeInput validates and sanitizes the input to prevent command injection.
+func sanitizeInput(input string) (string, error) {
```

Comment:
Why do we need to do this? They're running this on their own machine. 

Also if we are askibng for urls shouldnt' ths allow `:`

Reply:
true that was me fighting the pre-commit runner without questioning it